README.markdown

```diff
@@ -1,9 +1,17 @@
-# Sphero.js
+# 🚫 DEPRECATED Sphero.js
 
-The official Orbotix JavaScript SDK module to programmatically control Sphero robots.
+This was the official Orbotix JavaScript SDK module to programmatically control Sphero robots manufactured through mid-2016. **This repo is no longer supported or maintained.**
 
 [![Build Status](https://travis-ci.org/orbotix/sphero.js.svg?branch=master)](https://travis-ci.org/orbotix/sphero.js) [![Code Climate](https://codeclimate.com/github/orbotix/sphero.js/badges/gpa.svg)](https://codeclimate.com/github/orbotix/sphero.js) [![Test Coverage](https://codeclimate.com/github/orbotix/sphero.js/badges/coverage.svg)](https://codeclimate.com/github/orbotix/sphero.js/coverage)
 
+
+*New SDKs have been produced for the [Sphero RVR](https://www.sphero.com/rvr). Links to the SDKs can be found below and more information can be found at [sdk.sphero.com](https://sdk.sphero.com/)*
+- [micro:bit Python SDK](https://github.com/sphero-inc/sphero-sdk-microbit-python)
```

Comment:
I would also add the micro:bit MakeCode SDK (and probably mention that they can find the extension in the MakeCode editor as well)

Reply:
Oh! okay :D I didn't know that they could also use it from GitHub (it has been a minute since I've touched SDK stuff 🙈 ) Thanks for the tip!

Reply:
np! I'm wondering if we should also mention that the Arduino one is available in their Library Manager? and idk if any of the raspberry pi ones are available elsewhere... or we could just have the github links and make sure we mention in those READMEs if/where they can access the SDKs somewhere else